tools/cache.py

```diff
@@ -16,16 +16,6 @@
 
 # Permanent cache for system librarys and ports
 class Cache:
-  # If EM_EXCLUSIVE_CACHE_ACCESS is true, this process is allowed to have direct
-  # access to the Emscripten cache without having to obtain an interprocess lock
-  # for it. Generally this is false, and this is used in the case that
-  # Emscripten process recursively calls to itself when building the cache, in
-  # which case the parent Emscripten process has already locked the cache.
-  # Essentially the env. var EM_EXCLUSIVE_CACHE_ACCESS signals from parent to
-  # child process that the child can reuse the lock that the parent already has
-  # acquired.
-  EM_EXCLUSIVE_CACHE_ACCESS = int(os.environ.get('EM_EXCLUSIVE_CACHE_ACCESS', '0'))
```

Comment:
Why remove this?

Reply:
I don't follow. `emcc -c -sUSE_SDL2` will fetch SDL2 and write it into the cache, even though this is a compile-only command - we need to fetch SDL2 to get its header in order to compile with them?

Reply:
> I don't follow. `emcc -c -sUSE_SDL2` will fetch SDL2 and write it into the cache, even though this is a compile-only command - we need to fetch SDL2 to get its header in order to compile with them?

I that case the parent process is the only one that writes to the cache.  The parent triggers N different compile jobs, then it run `emar` to combine them, but neither the compile or the emar need to lock the cache.   Only the parent would lock the cache in this case.

Allowing the child process to modify the cache while the parent is hold the lock was always a little strange.

Reply:
In the past all the children would pointlessly/needlessly lock the cache just the run the sanity check.